src/containers/barray.mli

```diff
@@ -76,24 +76,23 @@ val mapi : (int -> 'a -> 'b) -> 'a array -> 'b array
     function is applied to the index of the element as first argument,
     and the element itself as second argument. *)
 
-val fold_left : ('a -> 'b -> 'a) -> 'a -> 'b array -> 'a
+val foldLeft : ('a -> 'b -> 'a) -> 'a -> 'b array -> 'a
 (** [Array.fold_left f x a] computes
     [f (... (f (f x a.(0)) a.(1)) ...) a.(n-1)],
     where [n] is the length of the array [a]. *)
 
-val fold_right : ('b -> 'a -> 'a) -> 'b array -> 'a -> 'a
+val foldRight : ('b -> 'a -> 'a) -> 'b array -> 'a -> 'a
 (** [Array.fold_right f a x] computes
     [f a.(0) (f a.(1) ( ... (f a.(n-1) x) ...))],
     where [n] is the length of the array [a]. *)
 
-external make_float: int -> float array = "caml_make_float_vect"
+external makeFloat: int -> float array = "caml_make_float_vect"
```

Comment:
do these specialized arrays make sense in JS? Are they compiled to JS TypedArrays?